config/locales/telephony/fr.yml

```diff
@@ -17,7 +17,7 @@ fr:
         %{app_name} : Votre code à usage unique est %{code}. Il expire dans %{expiration} minutes. Ne partagez ce code avec personne.
 
         @%{domain} #%{code}
-      voice: Bonjour ! Votre code à usage unique de %{format_length} %{format_type} pour %{app_name} est %{code}. Votre code à usage unique est %{code}. De nouveau, votre code à usage unique est %{code}. Ce code expire dans %{expiration} minutes.
+      voice: Bonjour ! Votre code de %{format_length} %{format_type} à usage unique pour %{app_name} est %{code}. Votre code à usage unique est %{code}. À nouveau, votre code à usage unique est %{code}. Ce code expire dans %{expiration} minutes.
```

Comment:
Is this space (now a different space character?) between "Bonjour" and "!" an intentional inflection thing?

Reply:
Yes, this is how French punctuation goes!

Reply:
TIL! Thanks.